### PR TITLE
Provide new envs performance in node's metadata

### PR DESCRIPTION
**Pull request by Krigpl**
And that'll automatically make the netmasking use proper values for new envs.

### CODE REVIEW

**Review by maaktweluit — APPROVED** [review]
LGTM!

Should we also update this `p2pservice.metadata` after the benchmarks are ran?

**Comment by Krigpl** [comment]
@maaktweluit These are metadata providers meaning that they are invoked everytime metadata is needed. So after rerunning the benchmarks they will return most recent values too.

**Comment by maaktweluit** [comment]
@Krigpl Thanks for the clarification, it indeed gets the latest data this way

File: golem/client.py [hunk 1]
```diff
@@ -385,6 +383,18 @@ def start_network(self):
             task_finished_cb=self._task_finished_cb,
         )
 
+        def get_performance_values():
+            values = self.environments_manager.get_performance_values()
+            new_env_manager = self.task_server.task_keeper.new_env_manager
+            for new_env in new_env_manager.environments():
+                env_id = new_env.metadata().id
+                value = new_env_manager.get_cached_performance(env_id)
+                if value:
```
Comment (Wiezzel) [inline golem/client.py]:
```suggestion
                if value is not None:
```
Performance might be `0.0` and we want to include it then.

**Review by Wiezzel — COMMENTED** [review]
Looks OK, but what about this part?
>Redesign the way the benchmark scores are stored for the known peers. Create a separate column for benchmark scores to make it more explicit.

**Comment by Krigpl** [comment]
@Wiezzel I did not understand the goal of this part. Do we wanna a single column which still contains all the benchmarks of new evns? So still a json value? That's what we have now so there's no benefit to that.

**Review by Wiezzel — APPROVED** [review]

**Pull request merged**